.insomnia/Request/req_8f843e785e1c442d961a9ed0437d289c.yml

```diff
@@ -0,0 +1,37 @@
+_id: req_8f843e785e1c442d961a9ed0437d289c
+type: Request
+parentId: fld_80bc7ee1b14547dc860545b5af4878c5
+modified: 1681910084632
+created: 1681907101878
+url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/autoresolve"
+name: Auto Resolve
+description: >-
+  This endpoint triggerrs an `autoresolve` job for the given review
```

Comment:
We can mention here that once the job is done, a Pusher event will be sent out to subscribers.

Reply:
Done.